go.mod

```diff
@@ -32,7 +32,7 @@ require (
 replace (
 	github.com/containernetworking/cni => github.com/containernetworking/cni v0.8.1
 	github.com/containers/buildah => github.com/containers/buildah v1.23.1
-	github.com/redhatinsights/yggdrasil => github.com/jakub-dzon/yggdrasil v0.0.0-20211012071055-27d969343f4e
+	github.com/redhatinsights/yggdrasil => ../yggdrasil
```

Comment:
should yggdrasil version be pinned to a specific one?

Reply:
yes, yes, as soon my PR are merged we'll pin to the stable version 0.3 :+1: